horizons/util/loaders/sqliteatlasloader.py

```diff
@@ -53,24 +53,13 @@ def init(self):
 			self.atlaslib.append(img)
 		self.inited = True
 
-	def loadResource(self, location):
+	def loadResource(self, id):
 		"""
-		@param location: String with the location. See below for details:
-		Location format: <animation_id>:<command>:<params> (e.g.: "123:shift:left-16, bottom-8)
-		Available commands:
-		- shift:
-		Shift the image using the params left, right, center, middle for x shifting and
-		y-shifting with the params: top, bottom, center, middle.
-		A param looks like this: "param_x(+/-)value, param_y(+/-)value" (e.g.: left-16, bottom+8)
-		- cut:
-		#TODO: complete documentation
+		@param id: String with the location.
```

Comment:
`id` is a built-in function. Could you use a different name?

Reply:
yep, `anim_id` id probably better and I should document it a bit better